utilities/cdpetron.py

```diff
@@ -144,27 +144,34 @@ def get_lists(branch_dir, language, config, test):
     return gendate
 
 
-def scrap_pages(branch_dir, language, dump_lang_dir, test):
-    """Get the pages from wikipedia."""
-    articles_dir = os.path.join(dump_lang_dir, DUMP_ARTICLES)
-    logger.info("Assure articles dir is empty: %r", articles_dir)
-    if os.path.exists(articles_dir):
-        shutil.rmtree(articles_dir)
-    os.mkdir(articles_dir)
+def _call_scrapper(branch_dir, language, dump_lang_dir, articles_file, test=False):
+    """Prepare the command and run scraper.py."""
```

Comment:
Me parece que quedó medio pobre el docstring. Además ahora noté que aca dejé una línea en blanco despues y en las otras dos funciones que toqué no. Debería normalizar eso?